### PR TITLE
Add support for nerdctl ps --quiet

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <smahajan@roblox.com>

/cc @AkihiroSuda

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
```
root@vagrant:~/go/src/github.com/nerdctl# nerdctl ps --all
CONTAINER ID    IMAGE                              COMMAND                   CREATED           STATUS     PORTS    NAMES
0f402be30749    docker.io/library/alpine:latest    "/bin/sh"                 33 hours ago      Created
644897ab5839    docker.io/library/ubuntu:18.04     "bash"                    17 minutes ago    Created
c596fdd3_hun    docker.io/library/redis:alpine     "docker-entrypoint.s…"    33 hours ago      Running
root@vagrant:~/go/src/github.com/nerdctl# nerdctl ps --all --quiet
CONTAINER ID    IMAGE    COMMAND    CREATED    STATUS    PORTS    NAMES
0f402be30749
644897ab5839
c596fdd3_hun
```

File: ps.go [hunk 1]
```diff
@@ -65,6 +70,7 @@ func psAction(clicontext *cli.Context) error {
 func printContainers(ctx context.Context, clicontext *cli.Context, containers []containerd.Container) error {
 	trunc := !clicontext.Bool("no-trunc")
 	all := clicontext.Bool("all")
+	quiet := clicontext.Bool("quiet")
 
 	w := tabwriter.NewWriter(clicontext.App.Writer, 4, 8, 4, ' ', 0)
 	fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
```
Comment (AkihiroSuda) [inline ps.go:76]: header needs to be hidden for quiet mode

Comment (shishir-a412ed) [reply ps.go:76]: Nice catch :) let me fix it.

File: README.md [hunk 2]
```diff
@@ -63,7 +63,8 @@ Also, `nerdctl` might be potentially useful for debugging Kubernetes clusters, b
 
 - `nerdctl ps`
   - `-a, --all`: Show all containers (default shows just running)
-  - `--no-trunc`
+  - `--no-trunc`: Don't truncate output
+  - `-q, --quiet`: Only display numeric IDs
```
Comment (AkihiroSuda) [inline README.md:67]: not numeric 😛 

`docker ps --help` describes `-q, --quiet`  as "Only display container IDs"

Comment (shishir-a412ed) [reply README.md:67]: Strange! I am on mac and this is what I see

```
SM-smahajan628:images smahajan$ docker ps --help

Usage:	docker ps [OPTIONS]

List containers

Options:
  -a, --all             Show all containers (default shows just running)
  -f, --filter filter   Filter output based on conditions provided
      --format string   Pretty-print containers using a Go template
  -n, --last int        Show n last created containers (includes all states) (default -1)
  -l, --latest          Show the latest created container (includes all states)
      --no-trunc        Don't truncate output
  -q, --quiet           Only display numeric IDs
  -s, --size            Display total file sizes
```

File: ps.go (hunk 1 above)
Comment (shishir-a412ed) [reply ps.go]: Fixed.

File: README.md (hunk 2 above)
Comment (shishir-a412ed) [reply README.md:67]: Lmk if you still want me to change it to `Only display container IDs`

Comment (AkihiroSuda) [reply README.md:67]: Interesting , may be your CLI is old?

image

Comment (AkihiroSuda) [reply README.md:67]: Seems changed in v20.10.0-beta1 https://github.com/docker/cli/commit/9d9568263fbaad7253354931f9b3d1b9446d0c12 .

"Only display numeric IDs" is the new and correct form.

Comment (shishir-a412ed) [reply README.md]: Fixed.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**